modules/installation-ibm-z-user-infra-machines-iso.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_z/installing-ibm-z.adoc
+
+[id="installation-user-infra-machines-iso-ibm-z_{context}"]
+= Creating {op-system-first} machines
+
+Before you install a cluster on IBM Z infrastructure that you provision, you must install {op-system} on z/VM guest virtual machines for the cluster to use. Complete the following steps to create the machines.
+
+.Prerequisites
+
+* An FTP server running on your provisioning machine that is accessible to the machines you create.
+
+.Procedure
+
+. Log in to Linux on your provisioning machine.
+
+. Download the Red Hat Enterprise Linux CoreOS installation files from the link:https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/4.2/latest/[{op-system} image mirror].
++
+[IMPORTANT]
+====
+The {op-system} images might not change with every release of {product-title}.
+You must download images with the highest version that is less than or equal
+to the {product-title} version that you install. Use the image versions
+that match your {product-title} version if they are available.
+====
++
+Download the following files:
+
+* The initramfs: `rhcos-<version>-installer-initramfs.img`
+* The kernel: `rhcos-<version>-installer-kernel`
+* The operating system image for the disk on which you want to install {op-system}. This type can differ by virtual machine:
++
+`rhcos-<version>-dasd.raw.gz` for DASD
```

Comment:
@zzoubkova, per https://github.com/jhradilek/openshift-docs/commit/53595164cede4ba7699bc02ee397567c6cd74bac#r36452811, this should be  `rhcos-<version>-s390x-metal-dasd.raw.gz`

Reply:
Fixed in 36c8aa640b1c0301f102b11713b57ea33b720242